### PR TITLE
Update leonardo-client_2.13 from 1.3.6-8740f99-SNAP to 1.3.6-7579402

### PR DESCRIPTION
**Pull request by broadbot**
## About this PR
📦 Updates org.broadinstitute.dsde.workbench:leonardo-client_2.13 from `1.3.6-8740f99-SNAP` to `1.3.6-7579402`

## Usage
✅ **Please merge!**

I'll automatically update this PR to resolve conflicts as long as you don't change it yourself.

If you'd like to skip this version, you can just close this PR. If you have any feedback, just mention me in the comments below.

Configure Scala Steward for your repository with a [`.scala-steward.conf`](https://github.com/scala-steward-org/scala-steward/blob/b83aae55d9dd000548c3b3c9b63d79636e7b3c8b/docs/repo-specific-configuration.md) file.

_Have a fantastic day writing Scala!_

<details>
<summary>⚙ Adjust future updates</summary>

Add this to your `.scala-steward.conf` file to ignore future updates of this dependency:
```
updates.ignore = [ { groupId = "org.broadinstitute.dsde.workbench", artifactId = "leonardo-client_2.13" } ]
```
Or, add this to slow down future updates of this dependency:
```
dependencyOverrides = [{
  pullRequests = { frequency = "30 days" },
  dependency = { groupId = "org.broadinstitute.dsde.workbench", artifactId = "leonardo-client_2.13" }
}]
```
</details>

<sup>
labels: library-update, early-semver-pre-release, semver-spec-pre-release, commit-count:1
</sup>

### CODE REVIEW

**Review by cahrens — APPROVED** [review]

File: project/Dependencies.scala [hunk 1]
```diff
@@ -132,7 +132,7 @@ object Dependencies {
   val billingProfileManager = excludeJakarta("bio.terra" % "billing-profile-manager-client" % "0.1.175-SNAPSHOT")
   val terraCommonLib = tclExclusions(excludeJakarta("bio.terra" % "terra-common-lib" % "0.0.93-SNAPSHOT" classifier "plain"))
   val sam: ModuleID = excludeJakarta("org.broadinstitute.dsde.workbench" %% "sam-client" % "0.1-d606036")
-  val leonardo: ModuleID = "org.broadinstitute.dsde.workbench" % "leonardo-client_2.13" % "1.3.6-8740f99-SNAP"
+  val leonardo: ModuleID = "org.broadinstitute.dsde.workbench" % "leonardo-client_2.13" % "1.3.6-7579402"
```
Comment (cahrens) [inline project/Dependencies.scala:135]: @Qi77Qi why did you update this to a SNAP version? Any concerns about going to this version?

Comment (Qi77Qi) [reply project/Dependencies.scala:135]: the snap is from https://github.com/DataBiosphere/leonardo/pull/3565

We need to get WDS, cromwell etc to use the snap client from that PR first and upgrade all existing apps to use that snap client before we can merge that PR

**Review by marctalbott — APPROVED** [review]

**Comment by Qi77Qi** [comment]
can we close this for now if this is just auto update and no specific reason for the upgrade?

make me nervous having to track the client might've been reverted once I get back to pushing https://github.com/DataBiosphere/leonardo/pull/3565 through

File: project/Dependencies.scala (hunk 1 above)
Comment (cahrens) [reply project/Dependencies.scala:135]: OK, I will close this PR. But once you are are done with your work, can you update Rawls to a non-snap version so it is clear to the Rawls maintenance team that your work is complete?

**Comment by cahrens** [comment]
Closing per request from @Qi77Qi

File: project/Dependencies.scala (hunk 1 above)
Comment (Qi77Qi) [reply project/Dependencies.scala:135]: Will do!

**Pull request closed without merging**